### PR TITLE
💄(layout) add default image for empty category, organization and person

### PR DESCRIPTION
**Pull request by sveetch**
## Purpose

It is required to use some default images for empty content instead
of basic texts.

## Proposal

This commit adds default images for person portrait, category,
organization logo and organization banner. To do so, it adjusts person
 and organization detail and glimpse templates, and finally their CSS.

### CODE REVIEW

File: src/richie/apps/courses/templates/courses/cms/fragment_organization_glimpse.html [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+{% load i18n cms_tags extra_tags static thumbnail %}
 {% comment %}Obviously, the context template variable "organization" is required and must be an Organization page extension{% endcomment %}
 
 {% with organization_page=organization.extended_object organization_variant=organization_variant|default:"glimpse" %}
 {% if organization_variant == "row" %}
     <a class="organization-{{ organization_variant }}{% if organization_page.publisher_is_draft is True %} organization-{{ organization_variant }}--draft{% endif %}" href="{{ organization_page.get_absolute_url }}" title="{{ organization_page.get_title }}">
         <div class="organization-{{ organization_variant }}__logo">
             {% get_page_plugins "logo" organization_page as plugins or %}
-                <p class="organization-{{ organization_variant }}__empty">{% trans "Logo" %}</p>
+                <img src="{% static "richie/images/empty/organization_logo.png" %}"
```
Comment (sampaccoud) [inline src/richie/apps/courses/templates/courses/cms/fragment_organization_glimpse.html:9]: you need to use thumbnails with srcset as our other images for responsiveness.

**Comment by sveetch** [comment]
This is pending PR to merge once real empty image for organization logo have been given, the first one which have it can put it (optimized) then merge.

File: src/richie/apps/courses/templates/courses/cms/fragment_person_glimpse.html [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
+{% load i18n cms_tags extra_tags static thumbnail %}{% spaceless %}
 {% comment %}Obviously, the context template variable "person" is required and must be a Person page extension{% endcomment %}
 {% with person_page=person.extended_object %}
 <div class="person-glimpse">
     {% comment %}Use tabindex and aria-hidden on the image link as it is entirely redundant with the title link{% endcomment %}
     <a class="person-glimpse__media" href="{{ person_page.get_absolute_url }}" tabindex="-1" aria-hidden="true">
         {% get_page_plugins "portrait" person_page as plugins or %}
-            <p class="person-glimpse__media__empty">{% trans "Portrait" %}</p>
+            <img src="{% static "richie/images/empty/person_portrait.png" %}"
+                 class="person-glimpse__media__empty"
```
Comment (sampaccoud) [inline src/richie/apps/courses/templates/courses/cms/fragment_person_glimpse.html:9]: should be --empty like elsewhere?

File: src/richie/apps/courses/templates/courses/cms/person_detail.html [hunk 3]
```diff
@@ -44,8 +44,10 @@ <h1 class="subheader__title">{% render_model current_page "title" %}</h1>
                 <div class="subheader__aside">
                     <div class="subheader__cartouche">
                         {% get_placeholder_plugins "portrait" as plugins or %}
-                        <div class="subheader__empty">
-                        {% trans "Portrait" %}
+                        <div class="subheader__media subheader__media--locket subheader__empty">
+                            <img src="{% static "richie/images/empty/person_portrait.png" %}"
+                                 class="subheader__empty--avatar"
```
Comment (sampaccoud) [inline src/richie/apps/courses/templates/courses/cms/person_detail.html:49]:
```suggestion
                                 class="subheader__avatar--empty"
```

Comment (sveetch) [reply src/richie/apps/courses/templates/courses/cms/person_detail.html]: Done

File: src/richie/apps/courses/templates/courses/cms/fragment_person_glimpse.html (hunk 2 above)
Comment (sveetch) [reply src/richie/apps/courses/templates/courses/cms/fragment_person_glimpse.html]: Done

**Review by sampaccoud — APPROVED** [review]

**Pull request merged**